ContextDesktop/invalid

```diff
@@ -0,0 +1,4 @@
+<scores>
+</scores>
+<scores>
+</scores>
```

Comment:
Whoops, that was a file that was generated in an earlier test. Will remove before PR.